projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.ts

```diff
@@ -185,4 +186,20 @@ export class QuoteReviewComponent implements OnInit, OnDestroy {
   isEditable(code: string): boolean {
     return code !== BindingStateType.BIND;
   }
+
+  setBindingState() {
+    this.subscription.add(
+      this.cart$
+        .pipe(
+          tap(cart => {
+            if (
+              (<FSCart>cart).insuranceQuote.state.code === BindingStateType.BIND
+            ) {
+              localStorage.setItem('bindingState', 'true');
```

Comment:
We shouldn't access localStorage directly like this, rather through Spartacus' WindowRef, like so:
`this.winRef.localStorage.setItem('bindingState', 'true')`